lib/actions/ui.js

```diff
@@ -168,7 +168,14 @@ export function setViewedNearbyCoords(payload) {
       'nearby',
       [payload.lat, payload.lon].join(',')
     )
-    dispatch(routeTo(path, payload.id ? `?entityId=${payload.id}` : undefined))
+    dispatch(
+      routeTo(
+        path,
+        payload.gtfsId || payload.id
```

Comment:
Not holding this against this PR, but the search is getting cleared as a result of passing this second param. Although using the browser back navigation command does restore the search params, the UI is not picking those params. I'll file this as a separate issue.